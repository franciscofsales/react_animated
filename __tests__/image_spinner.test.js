import 'react-native';
import React from 'react';
import {ImageSpinner} from '../';

import renderer from 'react-test-renderer';
it('renders correctly', () => {
  const tree = renderer.create(
    <ImageSpinner />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});