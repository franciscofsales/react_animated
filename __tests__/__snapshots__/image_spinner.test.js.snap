exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 25,
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
      }
    }>
    <View
      style={
        Object {
          "height": 32,
          "width": 32,
        }
      }>
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
    </View>
    <View
      style={
        Object {
          "height": 160,
          "justifyContent": "space-between",
          "width": 32,
        }
      }>
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
    </View>
    <View
      style={
        Object {
          "justifyContent": "space-between",
          "width": 32,
        }
      } />
    <View
      style={
        Object {
          "height": 224,
          "justifyContent": "space-between",
          "width": 32,
        }
      }>
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
    </View>
    <View
      style={
        Object {
          "justifyContent": "space-between",
          "width": 32,
        }
      } />
    <View
      style={
        Object {
          "height": 160,
          "justifyContent": "space-between",
          "width": 32,
        }
      }>
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
    </View>
    <View
      style={
        Object {
          "height": 32,
          "width": 32,
        }
      }>
      <Image
        source={1}
        style={
          Object {
            "height": 32,
            "transform": Array [
              Object {
                "rotate": "360deg",
              },
            ],
            "width": 32,
          }
        } />
    </View>
  </View>
</View>
`;
